assets/src/components/propertiesPanel/vehiclePropertiesPanel.tsx

```diff
@@ -198,15 +199,25 @@ const StatusContent = ({ selectedVehicle }: { selectedVehicle: Vehicle }) => (
   </>
 )
 
-const VehiclePropertiesPanel = ({ selectedVehicle }: Props) => {
-  const [tabMode, setTabMode] = useState<TabMode>("status")
+const VehiclePropertiesPanel = ({
+  selectedVehicle,
+  initialTab = "status",
+}: Props) => {
+  const [tabMode, setTabMode] = useState<TabMode>(initialTab)
```

Comment:
I opted to factor out `initialTab` for uniformity across testing, I can instead write a special case for the status tab if this is unwanted.

Reply:
I like this!